### PR TITLE
add new flags for identity engine

### PR DESCRIPTION
**Pull request by volodymyrmakarchuk-okta**
## Description:
- **What's changed?** 
- **Is this PR related to a Monolith release?** 

### Resolves:

* [OKTA-363507](https://oktainc.atlassian.net/browse/OKTA-363507)
* [OKTA-374183](https://oktainc.atlassian.net/browse/OKTA-374183)

ga

### CODE REVIEW

**Comment by jakubvul-okta** [comment]
Could you include a screenshot?

**Comment by volodymyrmakarchuk-okta** [comment]
> Could you include a screenshot?

sure
ie-label

File: packages/@okta/vuepress-theme-prose/global-components/ApiLifecycle.vue
Comment (francoislehoux-okta) [inline packages/@okta/vuepress-theme-prose/global-components/ApiLifecycle.vue:39]: Should these be set as `const DEFAULT_LINK = "/docs/reference/releases-at-okta/"` for example ?

File: packages/@okta/vuepress-theme-prose/global-components/ApiLifecycle.vue
Comment (shawyu-okta) [inline packages/@okta/vuepress-theme-prose/global-components/ApiLifecycle.vue:12]: I guess you're following the old pattern here but it would be great if we could use `===` and also check equality with constants rather than the string literals here

Comment (volodymyrmakarchuk-okta) [reply packages/@okta/vuepress-theme-prose/global-components/ApiLifecycle.vue:12]: about constants seems it not possible to `labelType` inside html, it undefined there, so or to use it inside `data` or just like string

Comment (shawyu-okta) [reply packages/@okta/vuepress-theme-prose/global-components/ApiLifecycle.vue:12]: I think it would be worth exploring one of [these solutions](https://coderethinked.com/3-different-ways-to-access-constants-in-a-vue-template/). I like the "Create private variables in created() hook" one the best.

This is definitely a point in favor of React over Vue 😄

File: packages/@okta/vuepress-theme-prose/global-components/ApiLifecycle.vue
Comment (shawyu-okta) [inline packages/@okta/vuepress-theme-prose/global-components/ApiLifecycle.vue:42]: Can we fix this typo to `EARLY_ACCESS`?

File: packages/@okta/vuepress-theme-prose/global-components/ApiLifecycle.vue
Comment (shawyu-okta) [inline packages/@okta/vuepress-theme-prose/global-components/ApiLifecycle.vue:20]: Just a placeholder reminder to get this link before we merge the PR

**Review by shawyu-okta — APPROVED** [review]
LGTM once the link is populated

File: packages/@okta/vuepress-theme-prose/global-components/ApiLifecycle.vue
Comment (shawyu-okta) [inline packages/@okta/vuepress-theme-prose/global-components/ApiLifecycle.vue:47]: Could we update these labels to all be similar in style? e.g. make this one `limited_ga` or update the others to be more verbose too like `ie` -> `Identity Engine` etc.

File: packages/@okta/vuepress-theme-prose/global-components/ApiLifecycle.vue
Comment (susanharper-okta) [inline packages/@okta/vuepress-theme-prose/global-components/ApiLifecycle.vue:13]:
```suggestion
      Identity engine
```
this is the agreed upon capitalization when referring to the Identity engine. FYI

File: packages/@okta/vuepress-theme-prose/global-components/ApiLifecycle.vue
Comment (susanharper-okta) [reply packages/@okta/vuepress-theme-prose/global-components/ApiLifecycle.vue:47]: -> Identity engine

**Review by susanharper-okta — APPROVED** [review]
just need to adjust Identity Engine to Identity engine and we're good.

**Pull request merged**
